python-perfplot/list_files.py

```diff
@@ -0,0 +1,66 @@
+import os
+import pathlib
+import shutil
+
+import perfplot
+
+
+def make_dir_with_files(dir_name, number_of_files):
+    flat_dir = pathlib.Path(dir_name)
+
+    if flat_dir.exists():
+        shutil.rmtree(flat_dir)
+
+    flat_dir.mkdir(exist_ok=True)
+
+    for file_number in range(number_of_files):
+        pathlib.Path(flat_dir / pathlib.Path(f"{file_number}.txt")).touch()
+    return flat_dir
+
+
+def glob_star(dir):
+    return list(dir.glob("*"))
+
+
+def glob_star_star(dir):
+    return list(dir.glob("**/*"))
+
+
+def rglob_star(dir):
+    return list(dir.rglob("*"))
+
+
+def iterdir_(dir):
```

Comment:
I don't think we need the trailing underscore character here since `iterdir` isn't a built-in name.